controllers/cloud.redhat.com/providers/logging/null.go

```diff
@@ -0,0 +1,33 @@
+package logging
+
+import (
+	crd "cloud.redhat.com/clowder/v2/apis/cloud.redhat.com/v1alpha1"
+	"cloud.redhat.com/clowder/v2/controllers/cloud.redhat.com/config"
+	p "cloud.redhat.com/clowder/v2/controllers/cloud.redhat.com/providers"
+)
+
+type NullLoggingProvider struct {
+	p.Provider
+	Config config.LoggingConfig
+}
+
+func (a *NullLoggingProvider) Configure(c *config.AppConfig) {
+	c.Logging = a.Config
+}
+
+func NewNullLogging(p *p.Provider) (LoggingProvider, error) {
+	provider := NullLoggingProvider{Provider: *p}
+
+	return &provider, nil
+}
+
+func (a *NullLoggingProvider) SetUpLogging(app *crd.ClowdApp) error {
+	a.Config = config.LoggingConfig{}
```

Comment:
Do you think it's more idiomatic to set all the fields within the struct literal? e.g. `config.LoggingConfig{Cloudwatch: cw}`

Reply:
sure thing